contributors/command-help.md

```diff
@@ -15,22 +15,50 @@ Supported Repositories:
 
 #### Description
 
-Build the binary based on the PR for testing.
+Build the binary based on the PR for testing. It also reruns the following
+continuous integration(CI) test cases:
+
+* `idc-jenkins-ci-tidb/unit-test`
+* `idc-jenkins-ci-tidb/build`
+* `idc-jenkins-ci-tidb/build_check_race`
+* `idc-jenkins-ci-tidb/check_dev`
+* `idc-jenkins-ci-tidb/check_dev_2`
 
 ### `/run-all-tests`
 
 #### Description
 
-Triggers all the CI test cases.
+Triggers all the CI test cases. Here is the list of all the CI test cases:
+
```

Comment:
/run-all-tests will not run build/build_check_race/check_dev/check_dev_2
and only tidb repo need mybatis-test/sqllogic-test etc. did it ok?

Reply:
Addressed, PTAL again. Thanks.